examples/sfml/tilesheet.rs

```diff
@@ -0,0 +1,73 @@
+use std::path::Path;
+
+use sfml::{
+    graphics::{FloatRect, IntRect, Texture},
+    SfBox,
+};
+use tiled::tileset::Tileset;
+
+/// A container for a tileset and the texture it references.
+pub struct Tilesheet {
+    texture: SfBox<Texture>,
+    tileset: Tileset,
+}
+
+impl Tilesheet {
+    /// Create a tilesheet from a Tiled tileset, loading its texture along the way.
+    pub fn from_tileset<'p>(tileset: Tileset) -> Self {
+        let tileset_image = tileset.image.as_ref().unwrap();
+
+        let texture = {
+            let texture_path = Path::new(&tileset_image.source);
+
+            Texture::from_file(texture_path.to_str().expect("obtaining valid UTF-8 path")).unwrap()
+        };
+
+        Tilesheet { texture, tileset }
+    }
+
+    pub fn texture(&self) -> &Texture {
+        &self.texture
+    }
+
+    pub fn tile_rect(&self, gid: u32) -> Option<IntRect> {
```

Comment:
Since we're building an SFML example here, wouldn't it be better to just directly return a `FloatRect`, so we can remove the separate `tile_uv` function?